spec/Section 3 -- Type System.md

```diff
@@ -327,18 +327,14 @@ represent additional fields a GraphQL client only accesses locally.
 ScalarTypeDefinition : Description? scalar Name Directives[Const]?
 
 Scalar types represent primitive leaf values in a GraphQL type system. GraphQL
-responses take the form of a hierarchical tree; the leaves on these trees are
+responses take the form of a hierarchical tree; the leaves of this tree are
```

Comment:
scalars, Enums or null values are the leaves of the tree

Reply:
let me think more about how to represent this idea.

I know we're not super consistent in our definition of scalar - sometimes we include enum and null as "scalar" and sometimes we do not.

Reply:
@andimarek @leebyron I think it's better to have separate PR that address this problem through the entire spec text to make it easy to review history of spec changes.

Reply:
Circling back on this, I'll make a small non-normative change here to resolve this (essentially just reference Enums). Agreed it needs a holistic editing pass to ensure "scalar" isn't ambiguous